yelp_package/dockerfiles/itest/api/marathon.json

```diff
@@ -1 +1,30 @@
-{"marathon_config": {"url": "http://marathon:8080", "password": null, "user": null}}
+{
+  "marathon_config": {
+    "url": "http://marathon:8080",
```

Comment:
One serious downside to this: the urls generated by this tool won't be "actually" useful to developers :(

The "real" urls for developers are yelpy, and change from dev/prod, etc. Any ideas on how to provide "real" urls?

I was thinking if we could have a magic function that did that, then it could power this dashboard *and* enable a clickable url in paasta status.

Reply:
My original plan was to do that at the UI layer (which will be a separate CR) and have some sort of map from these urls to the "real" urls. But after thinking about this some more, I guess I could put the map in the system paasta configs and have these scripts use those URLs instead.